app/models/rubric_criterion.rb

```diff
@@ -25,15 +25,16 @@ class RubricCriterion < Criterion
 
   has_many :test_groups, as: :criterion
 
+  DEFAULT_MAX_MARK = 4
```

Comment:
DEFAULT MAX MARK is currently being used in the new criterion erb file as a default mark for the maximum. I think this should be the only constant we should keep

Reply:
Okay we can keep this.